src/scss/components/_media-element.scss

```diff
@@ -0,0 +1,132 @@
+/**
+ * Media element
+ * ===================================
+ * Example – SERP – Restaurant logos
+ *
+ * A common pattern that sees an image followed by a block of information.
+ *
+ */
+
+$mediaElement-img-width: 50px;
+
+$mediaElement-img-width--mid: 78px;
+
+$mediaElement-img-width--wide: 67px;
+
+
+// mediaElement module
+// Usual of having an image to the left and content to the right
```

Comment:
This line doesn't read quite right.  Maybe should be `Usually consists of having…`

Reply:
It's a direct copy from INT CW, I can change it if you want but can we still aim to get this in today?

Reply:
Actually, I completely agree even if it is a cut and paste :laughing: